### DISCUSSION TITLE
Complete the expression

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#814] [Galaxy Trivia] [answered]
> I find 📺 very educational. The ⌚🧑 turns it on, I ...

**Bonus:** Name the author.

**Comment by ParanoidUser** [comment]
@Ju-yeop / @Producks, any ideas? 😶‍🌫️

**Comment by peterbaccaro** [comment, marked as answer]
“I find television very educational. Every time someone switches it on I **go into another room and read a good book**”.

This is a humorous quote by American comedian and actor **Groucho Marx**. It highlights his perspective on television and humorously suggests that he finds reading a good book more educational and enjoyable than watching television.

**Comment by SANKARAMDAS** [comment]
"I find television very educational. Every time someone switches it on I go into another room and read a good book.”
 -- Groucho Marx
Groucho Marx became famous on Broadway before moving on to starring roles in Hollywood. His comical skills and adaptability also allowed him to master radio and television. Yet, reportedly one of his sharpest remarks playfully disparaged TV:
